test/cases/api.sh

```diff
@@ -1386,7 +1376,6 @@ function verifyInGCP() {
 
   # create the instance
   # resource ID can have max 62 characters, the $GCP_TEST_ID_HASH contains 56 characters
-  GCP_INSTANCE_NAME="vm-$GCP_TEST_ID_HASH"
```

Comment:
This should no have been deleted, since the value is used below for creating a VM. The name is intentionally predictable to allow for cleanup in case of pass or fail.